be/src/storage/schema_change_utils.cpp

```diff
@@ -438,6 +450,34 @@ bool ChunkChanger::change_chunk_v2(ChunkPtr& base_chunk, ChunkPtr& new_chunk, co
     return true;
 }
 
+Status ChunkChanger::fill_materialized_columns(ChunkPtr& new_chunk) {
```

Comment:
`add column` will enter the linked schema change, it does not change the historical data.
I think it is not suitable for `add materialized column`, it should change the historical data.

Reply:
2. `mc_1 DOUBLE MATERIALIZED_COLUMN (array_avg(array_int))` this syntax seems a little strange.
3. What functions are supported in MATERIALIZED_COLUMN?